README.md

```diff
@@ -1,16 +1,58 @@
-This directory contains the programs needed to generate and verify SigStore root keys and create signed TUF metadata. 
+This directory contains the programs needed to generate and verify Sigstore root keys and create signed TUF metadata. 
+
+## TUF Repository Structure
+
+The current published repository metadata lives in the [repository](/repository/repository) subfolder of this GitHub repository. In this repository, you will find the top-level TUF metadata files, delegations, and target files. 
+
+* [root.json](repository/repository/root.json): This is the current `root.json`. It is signed by at least 3 out of the 5 [current root keyholders](https://github.com/sigstore/root-signing#current-sigstore-root-keyholders). Other signing keys endorsed by this root include:
+  * A set of 5 targets keyholders (the root keyholders).
+  * An online [snapshotting key](https://github.com/sigstore/root-signing/blob/57ac5cd83b90ff97af78db846eea2525eb0eee81/repository/repository/root.json#L87-L97) located at `projects/project-rekor/locations/global/keyRings/sigstore-root/cryptoKeys/snapshot`. You can verify the hex-encoded public key value with the following code:
```

Comment:
Given that project isn't accessible publicly, can we just say it's stored in GCP? Its exact location can be documented on an internal repository with playbooks.

Reply:
Honestly, shouldn't those root keys have public GetPublicKey view access?

Reply:
That's an interesting question, probably should bring it up in the public instance repo. While this project isn't secret, I'm not sure I want to start opening up resources from it. The bucket is publicly accessible, but that doesn't contain a reference to the project.

Reply:
Noted, I'll leave it out for now, and in the next root signing we should consider it I think, or at least publishing the public key information somewhere else (e.g. committing to it during a TUF signing event)

Reply:
I think I'll document the exact locations, they're in the snapshot workflow anyway.

Reply:
Fair point!